killprocess.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I think it'd be neater if we made a `bin/` directory at the top level of TorQ directory to hold executable scripts, like this one & the launchprocess one (just so we're not cluttering the top level of the repo with these scripts & any others in future)